opal-core-ws/src/main/java/org/obiba/opal/web/project/ProjectResource.java

```diff
@@ -133,14 +133,7 @@ public KeyStoreResource getKeyStoreResource() {
   @Path("/vcf-store")
   public VCFStoreResource getVCFStoreResource() {
     Project project = getProject();
-    if (!opalRuntime.hasServicePlugins()) throw new NoSuchServiceException(VCFStoreService.SERVICE_TYPE);
-    if (!project.hasVCFStoreService()) {
-      // for now get the first one. Some day, the service type will be a project admin choice
-      ServicePlugin service = opalRuntime.getServicePlugins()
-          .stream().filter(s -> opalRuntime.isVCFStorePluginService(s)).iterator().next();
-      project.setVCFStoreService(service.getName());
-      projectService.save(project);
-    }
+    if (!opalRuntime.hasServicePlugins() || !project.hasVCFStoreService()) throw new NoSuchServiceException(VCFStoreService.SERVICE_TYPE);
```

Comment:
This are different situations:
* there is no service => do not show the genotypes tab
* the project has no store service